regen_openapi.sh

```diff
@@ -1,19 +0,0 @@
-#!/bin/sh
-
-set -ex
-
-if [ -n "$1" ]; then
-    curl "$1" | python -m json.tool > openapi.json
-fi
```

Comment:
@svix-frank, was this being used?

Reply:
It shouldn't be, it has only caused confusion it leads new users to try and build from the latest openapi spec on the docs page which is no longer what should be done